src/test/kotlin/step2/LottosTest.kt

```diff
@@ -0,0 +1,25 @@
+package step2
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+import step2.domain.Lotto
+import step2.domain.Lottos
+
+class LottosTest {
+
+    @DisplayName("정답이 3개인 로또")
+    @Test
+    fun answerCountTest() {
```

Comment:
```suggestion
    @Test
    fun `정답이 3개인 로또`() {
```
코틀린에서는 위와 같이 함수명에 바로 한글을 사용할 수 있습니다.